CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ add_executable(SDLJumpStart
 	src/engine.cpp
 	src/font.cpp
 	src/main.cpp
+	src/sdl_utils.cpp
```

Comment:
The newline change creates too much noise to sneak in other stuff like this.
I just committed a fix for the new-lines, hadn't noticed because my editor just hid this.

The project is an old project that has kind of tumbled along over time, so it's got some cruft!

Anyway, please rebase from the newline commmit so we can look at these changes in better isolation.